src/utilities/transition/ExpandableTransition.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+// Required modules.
+var Events = require( '../../mixins/Events.js' );
+var BaseTransition = require( './BaseTransition' );
+var fnBind = require( '../function-bind' ).bind;
+var contains = require( '../dom-class-list' ).contains;
+
+// Exported constants.
+var CLASSES = {
+  BASE_CLASS:     'u-expandable-transition',
+  EXPANDED:       'u-expandable-expanded',
+  COLLAPSED:      'u-expandable-collapsed',
+  OPEN_DEFAULT:   'u-expandable-content__onload-open'
+};
+
+/**
+ * ExpandableTransition
+ * @class
+ *
+ * @classdesc Initializes new ExpandableTransition behavior.
+ *
+ * @param {HTMLNode} element
+ *   DOM element to apply move transition to.
+ * @returns {ExpandableTransition} An instance.
+ */
+function ExpandableTransition( element, CLASSES ) { // eslint-disable-line max-statements, no-inline-comments, max-len
+
+  var _baseTransition = new BaseTransition( element, CLASSES ),
+      timer,
+      isAnimating = false;
```

Comment:
Not sure if there's strong feelings around this, but the linter currently prefers defining each variable on it's own line like:
```
var _baseTransition = new BaseTransition( element, CLASSES );
var timer;
var isAnimating = false;
```

The thinking is that it's easier to move variables around when each is on its own line and also that multiple vars per line is easier to confuse with object literals when scanning the code, such as:

```
var _baseTransition = { new BaseTransition( element, CLASSES ),
    timer,
    isAnimating: false };
```